sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -44,13 +44,16 @@ internal static class DiagnosticVerifier
             "BC36716" // VB12 does not support line continuation comments" i.e. a comment at the end of a multi-line statement.
         };
 
-        public static void Verify(Compilation compilation, DiagnosticAnalyzer diagnosticAnalyzer, CompilationErrorBehavior checkMode)
+        public static void Verify(Compilation compilation, DiagnosticAnalyzer diagnosticAnalyzer, CompilationErrorBehavior checkMode) =>
```

Comment:
I've extended `DiagnosticVerifier.Verify` to run multiple analyzers in order to be able to keep tests for `InvalidCastToInterface` in the same file. Alternatively, if we decide this is not needed, we can split the tests and remove the changes done on `DiagnosticVerifier`.

Reply:
I think it's good to keep the test as is (one single test for two DiagnosticAnalyzers), especially for this refactoring PR

if we'll need to modify one of the DiagnosticAnalyzers in the future , we may want to be able to test each individually